README.md

```diff
@@ -108,7 +108,7 @@
 | Link  | Description  | Author | Language |
 |:------|:-------------|:-------|:---------|
 | [NovelCovid](https://npmjs.com/novelcovid) | Extremely fast and always up-to-date JavaScript API Wrapper (formerly covidapi). | [puf17640](https://github.com/puf17640) | English
-| [corona-api](https://pypi.org/project/corona-api) | Always up-to-date Python API Wrapper. | [apex2504](https://github.com/apex2504) | English
+| [disease.py](https://pypi.org/project/disease.py) | The official, up-to-date, and async-compatible Python API Wrapper (formerly corona-api).| [apex2504](https://github.com/apex2504) | English
```

Comment:
is it really the official python api wrapper? @Nekiono

Reply:
honestly i wasnt even aware myself it was made official until you said this on discord; https://i.takagi.computer/🥚😢🦎🐼🖌.png

Reply:
> honestly i wasnt even aware myself it was made official until you said this on discord; https://i.takagi.computer/🥚😢🦎🐼🖌.png

oh i actually said that hahaha, well, core team agreed on not having any official wrappers but having the "trusted" ones be "recommended wrappers"